pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -4,15 +4,27 @@ import (
 	"fmt"
 	"os"
 	"regexp"
+	"strconv"
 	"strings"
+	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
+	"github.com/openshift/library-go/pkg/operator/certrotation"
 	corev1 "k8s.io/api/core/v1"
 )
 
-const rewritePrefix = "rewritten.cert-info.openshift.io/"
+const (
+	rewritePrefix = "rewritten.cert-info.openshift.io/"
+	// Unit values in seconds
+	year   = 60 * 60 * 24 * 365
```

Comment:
please move this vars inside `durationToHumanReadableString` so that they are private to that method.

Reply:
good point, we would, but these var will only be used by that function and it would be nice to keep them inside that function for clarity. i don't think this code must be lightning fast. an alternative would be to create an internal pkg for that function.